src/composables/multisig.ts

```diff
@@ -13,12 +31,35 @@ interface UseMultisigOptions {
   store: Store<any>
 }
 
-/**
- * This composable provides methods related to multisig operations
- */
-export function useMultisig({ store }: UseMultisigOptions) {
+const POLLING_INTERVAL = 5000;
+const MULTISIG_TRANSCATION_EXPARATION_HEIGHT = 50;
+
+const LOCAL_STORAGE_MULTISIG_KEY = `${LOCAL_STORAGE_PREFIX}_multisig`;
+
+function storeMultisigAccounts(multisig: IMultisigAccount[], networkId: string) {
+  window.localStorage
+    .setItem(`${LOCAL_STORAGE_MULTISIG_KEY}_${networkId}`, JSON.stringify(multisig));
+}
+
+function getStoredMultisigAccounts(networkId: string): IMultisigAccount[] {
+  const storedMultisig = window.localStorage.getItem(`${LOCAL_STORAGE_MULTISIG_KEY}_${networkId}`);
+  return storedMultisig ? JSON.parse(storedMultisig) : [];
+}
+
+const multisig = ref(<IMultisigAccount[]>[]);
```

Comment:
Constant name "multisig" sounds too generic. I suggest to rename this to multisigVaults or multisigAccounts...